File: app/src/main/java/com/example/getfood/Activity/TermsActivity.java
package com.example.getfood.Activity;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.view.View;
import android.webkit.WebView;
import android.webkit.WebViewClient;

import com.example.getfood.R;

public class TermsActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        WebView wv = new WebView(getApplicationContext());
        setContentView(wv);

        wv.setVisibility(View.VISIBLE);
        wv.getSettings().setJavaScriptEnabled(true);
        wv.getSettings().setAllowFileAccess(true);
        wv.setWebViewClient(new TermsActivity.Callback());
        String working = getString(R.string.terms_working_url);

        String directdisplay = getString(R.string.terms_direct_url);
        wv.loadUrl(working);
    }

    private class Callback extends WebViewClient {
        @Override
        public boolean shouldOverrideUrlLoading(
                WebView view, String url) {
            return (false);
        }
    }
}

File: app/src/main/java/com/example/getfood/ui/terms/TermsActivity.kt
package com.example.getfood.ui.terms

import android.os.Bundle
import android.view.View
import android.webkit.WebView
import android.webkit.WebViewClient
import androidx.appcompat.app.AppCompatActivity
import com.example.getfood.R

class TermsActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val wv = WebView(applicationContext)
        setContentView(wv)
        wv.visibility = View.VISIBLE
        wv.settings.javaScriptEnabled = true
        wv.settings.allowFileAccess = true
        wv.webViewClient = Callback()
        val working = getString(R.string.terms_working_url)
        val directdisplay = getString(R.string.terms_direct_url)
        wv.loadUrl(working)
    }

    private inner class Callback : WebViewClient() {
        override fun shouldOverrideUrlLoading(
                view: WebView, url: String): Boolean {
            return false
        }
    }
}